CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 # Changelog
 
-## v1.1.0 - TBD
+## v1.2.0 - TBD
+
+## v1.1.0 - 2020-04
+
```

Comment:
I think you want to include the following here:

```
> **NOTICE:** This repo will be merging with the
[googleapis/google-cloud-cpp](https://github.com/googleapis/google-cloud-cpp)
monorepo in 60 days, around **June 1, 2020**. This change should not break any
users who are using a released version. We expect this new monorepo
organization to be better for our existing and new customers going forward.
Please see
[googleapis/google-cloud-cpp#3612](https://github.com/googleapis/google-cloud-cpp/issues/3612)
for more details.
```

This will match what we did in `-common`